package.json

```diff
@@ -23,37 +23,37 @@
     "node": ">=0.10.0"
   },
   "dependencies": {
-    "URIjs": "1.14.1",
-    "bluebird": "2.3.10",
+    "URIjs": "^1.14.2",
```

Comment:
The shrinkwrap should take care of it, no?

On Fri, Mar 27, 2015, 3:41 PM Vlad Filippov <notifications@github.com>
wrote:

> In package.json
> <https://github.com/mozilla/fxa-oauth-server/pull/237#discussion_r27336303>
> :
>
> > @@ -23,37 +23,37 @@
> >      "node": ">=0.10.0"
> >    },
> >    "dependencies": {
> > -    "URIjs": "1.14.1",
> > -    "bluebird": "2.3.10",
> > +    "URIjs": "^1.14.2",
>
> All these ^ should probably be removed. All other repos have this locked
> to avoid update surprises.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/fxa-oauth-server/pull/237/files#r27336303>.
>